deployment/installInsightAgent.py

```diff
@@ -108,8 +113,12 @@ def sshInstallHypervisor(retry,hostname):
     except socket.error, e:
         print "Socket connection failed in %s:"%hostname, e
         return sshInstallHypervisor(retry-1,hostname)
-    #except:
-        #print "Unexpected error in %s:"%hostname
+    except IOError,e :
+        print "Not enough disk space in host"
```

Comment:
My understanding is more than just Not enough disk space in host. For example, having no proper permission can cause IOError. Am I correct?

http://stackoverflow.com/questions/1134607/python-exception-handling

Can we print the error message here?